boot/setup.sh

```diff
@@ -208,6 +209,13 @@ sdc_log_rotation_add ufds-capi /var/svc/log/*ufds-capi*.log 1g
 sdc_log_rotation_add ufds-replicator /var/svc/log/*ufds-replicator*.log 1g
 sdc_log_rotation_setup_end
 
+# Setup crontab for expired credentials cleanup 
+echo "Setting up crontab for expired credentials cleanup"
+crontab -l 2>/dev/null | grep -v cleanup-expired-creds > /tmp/crontab.$$ || true
+echo "0 */12 * * * /opt/smartdc/$role/bin/cleanup-expired-creds -c /opt/smartdc/$role/etc/config.json >> /var/log/ufds-cleanup.log 2>&1" >> /tmp/crontab.$$
```

Comment:
I read this as a 12-hour cleanup interval, i.e. a cleanup pass every 12 hours. This means an expired temporary credential can have up to 11h59m59s of lifetime beyond its expiration.  Does this seem a bit wide, or is this acceptable?